src/js/consts/errors.js

```diff
@@ -9,12 +9,23 @@ export default {
   SegmentAppendError: 'segment-append-error',
   VttLoadError: 'vtt-load-error',
   VttCueParsingError: 'vtt-cue-parsing-error',
-  EMEKeySessionCreationError: 'eme-key-session-creation-error',
   // Errors used in contrib-ads:
   AdsBeforePrerollError: 'ads-before-preroll-error',
   AdsPrerollError: 'ads-preroll-error',
   AdsMidrollError: 'ads-midroll-error',
   AdsPostrollError: 'ads-postroll-error',
   AdsMacroReplacementFailed: 'ads-macro-replacement-failed',
-  AdsResumeContentFailed: 'ads-resume-content-failed'
+  AdsResumeContentFailed: 'ads-resume-content-failed',
+  // Errors used in contrib-eme:
+  EMEEncryptedError: 'eme-encrypted-error',
+  MSKeyError: 'ms-key-error',
+  WebkitKeyError: 'webkit-key-error',
```

Comment:
Could you elaborate more on this 3:

What are they?
Why do we need them?

```js
  EMEEncryptedError: 'eme-encrypted-error',
  MSKeyError: 'ms-key-error',
  WebkitKeyError: 'webkit-key-error',
```

Reply:
The idea behind these 3 was to be a catch-all for the outermost promise to the respective `handle` events. For `MSKeyError` and `WebkitKeyError` they specifically cover errors from legacy workflows ('msneedey' and 'webkitneedkey' events). Since this is not _really_ in-spec EME, I figured they were worthy of their own specific errors. For `EMEEncryptedError` it might not be necessary and was only added incase any other type of unknown error occurs and bubbles up to the `handleEncryptedEvent` promise.